third_party/xla/xla/service/gpu/BUILD

```diff
@@ -1568,7 +1568,7 @@ cc_library(
         "//xla/stream_executor:host_or_device_scalar",
     ]) + if_rocm_is_configured([
         "//xla/stream_executor/rocm:hipblas_lt_header",
-        "//xla/stream_executor/rocm:hipblaslt_plugin",
```

Comment:
may I ask what's purpose of changing this name?

Reply:
yes, this was needed as a workaround for rocm-6.0 platform, as it turns out, hipblas-lt tries to locate tensile library files by searching for a substring '*libhipblaslt*' and it happens to find it in
```
.../_solib_local/libxla_Sstream_Uexecutor_Srocm_Slibhipblaslt_Uplugin.so
```
as a result, tensile library files cannot be found. We have more detailed discussions of this in 'hipblaslt hotfix' chat